### PR TITLE
Modify candidate table

### PR DESCRIPTION
**Pull request by SaranchukVadym**

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 databaseChangeLog:
   - changeSet:
-      id: add-data-to-candidate-table
+      id: run-fix-table-candidate
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:3]: We cannot modify existing changesets, we should create a new one

File: sandbox-backend/src/main/resources/db/changelog/changes/sql/modify_candidate_table.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+ALTER TABLE candidate
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/sql/modify_candidate_table.sql:1]: Please change it to yaml file, cause in this case we can't add precondition

**Review by ikantsevich — APPROVED** [review]

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml (hunk 1 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:3]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/sql/modify_candidate_table.sql (hunk 2 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/sql/modify_candidate_table.sql:1]: corrected

**Pull request closed without merging**